healthyregions/SDOHPlace
# #264: make the "No results" match the new design

This PR addresses #264. It updates the "No results" section to reflect the new design, along with some style adjustments to the theme icons. Additionally, I updated the .gitignore file to include `nvmrc` for Node version control, as we're now using Node 18.

## How to Test
Run the app and search for a term (e.g., "a"). You should see the "No results" message appear with hard-coded theme suggestions. You can click the "X" in the search box to clear the search at any time:

<img width="1961" alt="Screenshot 2024-09-05 at 1 35 04 PM" src="https://github.com/user-attachments/assets/240c3954-8adb-4e3a-bf29-b40e30065a77">
